package.json

```diff
@@ -7,7 +7,8 @@
     "gh-pages": "^2.1.1"
   },
   "dependencies": {
-    "typedoc" : "0.15.0"
+    "typedoc" : "0.15.0",
+    "verdaccio": "4.3.3"
```

Comment:
verdacio is not needed. It is used during development for verifying the packages